src/database/database.service.js

```diff
@@ -0,0 +1,20 @@
+import {GuildModel} from "./models/guild.js";
+
+export class DatabaseService {
```

Comment:
I now get why and how you wanted to use the `GuildService` class and I agree.

Sorry for asking but would you be so kind to revert that to the `GuildService`

Thank you!

Reply:
Absolutely! Please test the new commit :)